templates/scylla-jmx.j2

```diff
@@ -0,0 +1,3 @@
+{% for key,value in scylladb_jmx|dictsort(by="key") %}
```

Comment:
Not sure if we want this key,value items.  I'm not opposed to it, but it's unclear what goes in here.  It's also nice to have defaults, stuff people will not need to touch (or should).  Is this file needed at was a question at the top related to the key val item.

Reply:
Removed